framework/wazuh/core/cluster/common.py

```diff
@@ -1705,6 +1706,9 @@ def send_data_to_wdb(data, timeout, info_type='agent-info'):
                 except TimeoutError as e:
                     raise e
                 except Exception as e:
+                    if any(ignored_exception in str(e) for ignored_exception in ignored_wdb_exceptions):
+                        continue
+
```

Comment:
Not sure if Python is optimizing this kind of scenarios but since we are using `str(e)` anyway I would create a variable for it and avoid converting the value twice.

```suggestion
                   error = str(e)
                    if any(ignored_exception in error for ignored_exception in ignored_wdb_exceptions):
                        continue

                    result['error_messages']['chunks'].append((i, error))
```